modules/virt-deleting-virt-cli.adoc

```diff
@@ -37,6 +37,13 @@ $ oc delete subscription kubevirt-hyperconverged -n openshift-cnv
 $ oc delete csv -n openshift-cnv -l operators.coreos.com/kubevirt-hyperconverged.openshift-cnv
 ----
 
+. Delete the {VirtProductName} namespace:
++
+[source,terminal]
+----
+$ oc delete namespace -n openshift-cnv
```

Comment:
I think the command should be: ` oc delete namespace openshift-cnv:

```
$  oc delete namespace -n openshift-cnv
error: resource(s) were provided, but no name was specified
$  oc delete namespace openshift-cnv
namespace "openshift-cnv" deleted
```

Reply:
I know it is outside of the scope of this PR. But, the [web console install instructions](https://docs.openshift.com/container-platform/4.13/virt/install/installing-virt-web.html) are a bit off. 

> 2. In the Filter by keyword field, type OpenShift Virtualization Operator.

The user needs to search for `OpenShift Virtualization`. If you search for `OpenShift Virtualization Operator`, nothing was coming up for me. 

![image](https://github.com/openshift/openshift-docs/assets/25209155/fd644364-e196-49d8-9d6a-2638f08a5913)